README.md

```diff
@@ -73,4 +116,24 @@ Gradle-Version: 7.3.3
 * `Implementation-Title` is your `project.name`
 * The `Implementation-Vendor` can be set by configuring the `vendorName` in the `JavaConfigurePluginExtension` (see above)
 
+## Auto-Configuration
+
+In multi-module projects, each of the above mentioned plugins can be configured on a per-module basis,
+but you can also use the `autoConfigure` extension of this plugin to set defaults for all submodules without
+iterating through all modules by yourself.
+
+To achieve that, add the following block to your root module `build.gradle` in a multi-module project:
+
+````groovy
+autoConfigure {
+    java {
+        languageVersion = JavaLanguageVersion.of(17)
+        encoding = "UTF-16"
+        vendorName = "My cool company"
+    }
+}
+````
 
+Each property here comes with an equivalent to the above mentioned sub-plugins. If you set a default value
```

Comment:
not really true in the case of `applicationBuild`

Reply:
we can add that later as well